API/users/tests.py

```diff
@@ -10,11 +13,31 @@ def setUp(self):
         self.user = User.objects.create_user("filip",password="kochamsiostre123")
         self.user.save()
 
+    def test_rsa(self):
+        """Check is RSA is enabled"""
+        from django.conf import settings
+        self.assertEqual(settings.SIMPLE_JWT["ALGORITHM"], "RS512")  
```

Comment:
when i said testrsa, I was talking about testing whether the resulting jwt has a correct algo in its header, this doesn't test if the JWTs are really signed using RS512

Reply:
So this test should generate the JWT manually using other means than the authentication library, take the JWT that the library generated, and then compare the two? Is that what you mean?

Reply:
No, what I meant was we can hit the app's token_obtain_pair endpoint and check the header of the SWT we receive. 

Every SWT has metadata in the first part of the token. You can view it by base64 decoding the part of the SWT up until the first '.'.

You can use the https://jwt.io/ site's debugger for some context, it let's you decode JWTs and do some other nice things interactively

Reply:
Okay, I understand. I will write a test which checks whether the algorithm in the header is indeed set to `RS512` and if the `typ` propery is set to `JWT`.